src/main/rst/03_webdriver.rst

```diff
@@ -703,6 +756,13 @@ People often wish to retrieve the innerText value contained within an element.
 
     element = driver.find_element_by_id("element_id")
     element.text
+
+.. code-block:: javascript
+	
+	var element = driver.findElement(By.id('elementID'));
+	element.getText().then(function(text) {
```

Comment:
var element = driver.findElement(By.id('elementID'));
    element.getText().then(text => console.log(`Text is ${text}`));